plinth/modules/minetest/__init__.py

```diff
@@ -107,17 +120,46 @@ def disable():
     frontpage.remove_shortcut('minetest')
 
 
-class MinetestServiceView(ServiceView):
-    service_id = managed_services[0]
-    diagnostics_module_name = "minetest"
-    description = description
-    show_status_block = True
-
-
 def diagnose():
     """Run diagnostics and return the results."""
     results = []
 
     results.append(action_utils.diagnose_port_listening(30000, 'udp4'))
 
     return results
+
+
+def get_max_players_value():
+    """Return the current Max Players value."""
+    aug = load_augeas()
+    value = aug.get(AUG_PATH + '/max_users')
+    return value
```

Comment:
Why not convert to int here? It seems more natural that this function should return a number, not a string.